VimeoNetworking/Sources/Models/VIMUser.m

```diff
@@ -33,6 +33,7 @@
 #import "VIMPreference.h"
 #import "VIMUploadQuota.h"
 #import "VIMUserBadge.h"
+#import <VimeoNetworking/VimeoNetworking-Swift.h>
```

Comment:
do we need this import here?  I feel like this is already being included for the whole framework somewhere.  Maybe in the pch?

Reply:
@mikew-personal I don't think so. I checked the `VIMVideo.m`, and that file needed this import to work properly with Swift classes. Should we create an `pch` file for `VimeoNetworking` framework?

Reply:
ok, if we're doing the same thing in VIMVideo already then I think this is fine.